doc/developers/logging_cpp.rst

```diff
@@ -67,7 +67,7 @@ Each function is installed as the handler for a specific HTTP resource, defined
     :end-before: SNIPPET_END: install_record
     :dedent: 6
 
-This example installs at ``"LOG_record", HTTP_POST``, so will be invoked for requests beginning ``POST /users/LOG_record``.
+This example installs at ``"LOG_record", HTTP_POST``, so will be invoked for requests beginning ``POST /app/LOG_record``.
```

Comment:
Let's drop the PREFIX_ stuff now (separate PR).

Reply:
Yes - I have a branch that mostly does this, just need to make it work for the lua and js apps as well.